### ISSUE TITLE
Buttons added through interactive Studio not appearing on the stream page

### ISSUE THREAD

**Issue by josurr** [DeekyJay/SoundwaveInteractive-releases#12]
So after being informed to use the Interactive Studio to edit the layout of the board, I added additional buttons to the layout through the Studio Editor Build page:

![interactive_studio_build](https://user-images.githubusercontent.com/14187881/28039736-39dff7f0-6578-11e7-8c00-8ac9ac03d0da.png)

The buttons do show up in the Soundwave Interactive desktop app:

![soundwave](https://user-images.githubusercontent.com/14187881/28039749-45e48c64-6578-11e7-82e2-697a08e6ef98.png)

However, the additional buttons do not show up on the stream page:

![stream_board](https://user-images.githubusercontent.com/14187881/28039780-5313aff0-6578-11e7-91b7-f22a22ed3ae5.png)

Any ideas?

**Comment by DeekyJay** [comment]
Any buttons that are unassigned will not show up below the stream.

**Comment by josurr** [comment]
@DeekyJay 
Good catch, however, I assigned sound files to all but the 3 top-most large buttons and they still do not all appear:

![asssigned_buttons](https://user-images.githubusercontent.com/14187881/28042814-d83f5778-6583-11e7-9141-bff85dfc212d.png)

**Comment by DeekyJay** [comment]
The text in the Interactive Studio get's overwritten, so you can't have titles floating around like that. I suggest having 3 dummy mp3 files to use as your title (or you could also make them actual sounds) and have them assigned so they show up.

**Comment by DeekyJay** [comment]
Also I noticed the ID's of your buttons are not numbered 0 through 29, so I'm actually surprised it works! Please let me know if you run into any issues! I'll reopen this issue for awhile and you can report them here. 😄

**Comment by josurr** [comment]
That's weird that the buttons have to be numbered. Is there a way to get them to say something other than "Unassigned" in the Studio?
